pybricksdev/connections.py

```diff
@@ -256,22 +256,34 @@ def nus_handler(self, sender, data):
         for line in lines:
             self.line_handler(line)
 
-    def pybricks_service_handler(self, _: int, data: bytes) -> None:
+    def __pybricks_service_handler(self, _: int, data: bytes) -> None:
         if data[0] == Event.STATUS_REPORT:
-            # decode the payload
-            (flags,) = struct.unpack("<I", data[1:])
-            program_running_now = bool(flags & Status.USER_PROGRAM_RUNNING.flag)
-
             # If we are currently downloading a program, we must ignore user
             # program running state changes, otherwise the checksum will be
             # sent to the terminal instead of being handled by the download
             # algorithm
-            if not self.loading:
-                if self.program_running != program_running_now:
-                    logger.info(f"Program running: {program_running_now}")
-                    self.program_running = program_running_now
-                if not program_running_now:
-                    self.user_program_stopped.set()
+            if not self.program_loaded:
+                logger.info("Program not yet loaded")
```

Comment:
This will be logged every 500 ms which seems a bit excessive. There are more use cases for this than just download and run where we may be connected for a long time without running a program.

Reply:
From what I've observed the default log level is `WARN`. I.e. these messages would be printed out only if `pybricksdev` is run with the `-d` parameter.

I don't mind removing it though.

Reply:
There are already bleak debug log messages that are triggered by the same event. Speaking more generally though, the `info` level should be considered something you might want to see in an an interactive command line program. In other words, the goal is to eventually replace use of `print()` in this library with `logger.info()` so that it can be used as a library in addition to a command line program.